stories/utils/allComponents.tsx

```diff
@@ -0,0 +1,19 @@
+import * as Card from '../../src/components/Card';
```

Comment:
all the components import should be done in `src/components/index.ts`
and this file should look like this:

```ts
import * as componentsScope from '../../src/components';
import * as iconsScope from 'wix-ui-icons-common';

export const LiveScope = {
  ...componentsScope,
  Icons: iconsScope,
};
```